### ISSUE TITLE
class accessLevels are not respected

### ISSUE THREAD

**Issue by hyyan** [BBx-Kitchen/bbj-language-server#74]
When any of the following rules are violated, an error or warning should be reported.

_The class access level must be specified, and must be either PUBLIC, PRIVATE, or PROTECTED.
A PRIVATE class is only visible to code within the same physical file.
A PUBLIC class can be accessed from code in other files.
A PROTECTED class (BBj 21.0 and higher) can only be accessed from programs within the same directory as the class or interface declaration._

https://documentation.basis.cloud/BASISHelp/WebHelp/commands/class_verb.htm

**Comment by StephanWald** [comment]
```
class public Test 

    method private doTest()
    methodend
classend

t! = new Test()
t!.d...
```
![image](https://github.com/user-attachments/assets/f8f1e690-bb52-4297-bb0a-19082716f7f2)

**Comment by StephanWald** [comment]
see also https://github.com/BBx-Kitchen/bbj-language-server/issues/171

**Comment by Lotes** [comment]
Closed with #195 ... for method's access levels I will create a new issue.
